### DISCUSSION TITLE
OpenAPI documentation plugin

### DISCUSSION THREAD

**Discussion by docs-of-all-trades** [fuma-nama/fumadocs#204] [Ideas]
Have you considered the option of an OpenAPI docs plugin, something like Docusaurus currently has?

(https://docusaurus-openapi.tryingpan.dev/)

**Comment by fuma-nama** [comment]
That's indeed an interesting idea, for now, you can generate MDX files from OpenAPI spec with existing CLI tools & libraries. 

I'm considering implementing a unified data source interface and the idea of source plugins. 
Welcome for contributions though, this feature might need to wait for the release of 7.0.0 I suppose.

**Comment by fuma-nama** [comment]
Supported via [OpenAPI Integration](https://fumadocs.vercel.app/docs/ui/openapi)
